app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ] Let's follow the project requirements on model relationships. On the ERD, a user is named an `author`. We can achieve this by explicitly adding a `foreign_key` called `author_id` in the user model.
hint: `belongs_to :author, class_name: 'User'`. Do not forget to do the same in user's model by including this line of code `has_many :comments, foreign_key: 'author_id'`. Kindly do the same to other relationships.